eq-author-api/schema/resolvers/base.js

```diff
@@ -907,22 +905,12 @@ const Resolvers = {
       return parentPage;
     },
     validation: answer =>
-      ["number", "date"].includes(getValidationEntity(answer.type))
+      ["number", "date", "dateRange"].includes(getValidationEntity(answer.type))
```

Comment:
We should probably make these constants

Reply:
This references values defined here: https://github.com/ONSdigital/eq-author-app/blob/39144650e141d176b0e66b37ab15c0cc0eb314ea/eq-author-api/utils/defaultAnswerValidations.js#L15 

Should we still convert to constants?